src/screens/Admin/screens/InstructorPulse/components/InstructorsList/components/InstructorSummary/index.js

```diff
@@ -4,6 +4,13 @@ import {viewActionText} from 'utils/text'
 import Heading from 'components/Heading'
 import Avatar from 'components/Avatar'
 
+const LessonCount = ({label, count}) => (
```

Comment:
We cool with small things like this inline as a const? Doesn't seem like it deserves a file.